src/datagrid/DataGrid.tsx

```diff
@@ -221,10 +241,16 @@ export enum SortOrder {
  * Enum for RowTpye :
  * @param {EXPANDABLE} for enabling expandable rows
  * @param {COMPACT} for enabling compact rows
+ * @param {ROWS_WITH_DETAIL_PANE} for enabling detail pane for rows
+ * @param {EXPANDABLE_ROWS_WITH_DETAIL_PANE} for enabling detail pane for expandable rows
+ * @param {COMPACT_ROWS_WITH_DETAIL_PANE} for enabling detail pane for compact rows
  */
 export enum GridRowType {
     EXPANDABLE = "expandable",
     COMPACT = "compact",
+    ROWS_WITH_DETAIL_PANE = "rows_with_detail_pane",
+    EXPANDABLE_ROWS_WITH_DETAIL_PANE = "expandable_rows_with_detail_pane",
+    COMPACT_ROWS_WITH_DETAIL_PANE = "compact_rows_with_detail_panes",
```

Comment:
This code suppose to be present in master branch. Can you please rebase and update PR diff ?

Reply:
https://github.com/EMCECS/clarity-react/pull/162  -> this PR will resolve this issue.  I cretaed branch from feature-master-beta3 and the feature-master-beta3 never merged to master.